Projects/TravelCertificate/Sources/TravelCertificateNavigation.swift

```diff
@@ -0,0 +1,148 @@
+import Flow
+import Foundation
+import Presentation
+import SwiftUI
+import hCore
+import hCoreUI
+
+public class TravelCertificateNavigationViewModel: ObservableObject {
+    public init() {}
+    @Published var isDocumentPresented: TravelCertificateModel?
+}
+
+struct TravelInsuranceSpecificationNavigationModel: Hashable {
+    public var id: String?
+    let specification: [TravelInsuranceContractSpecification]
+}
+
+enum TravelCertificateRouterActions: Hashable {
+    case whoIsTravelling(specifiction: TravelInsuranceContractSpecification)
+}
+
+enum TravelCertificateRouterActionsWithoutBackButton: Hashable {
+    case processingScreen
+}
+
+public struct TravelCertificateNavigation: View {
+    @StateObject private var travelCertificateNavigationVm = TravelCertificateNavigationViewModel()
+    @StateObject var router = Router()
+    private var canCreateTravelInsurance: Bool
+    private var infoButtonPlacement: ToolbarItemPlacement
+    private var openCoInsured: () -> Void
```

Comment:
not sure if there is a better way to do this?

Reply:
This seems fine.